app/gui2/src/components/visualizations/TableVisualization.vue

```diff
@@ -225,9 +225,19 @@ function formatText(params: ICellRendererParams) {
     '\t': '<span style="color: #df8800; white-space: break-spaces;">&#8594;  |</span>',
   }
 
+  const replaceSpacesNotInLinks = (text: string) => {
```

Comment:
This is complex. Why not replace spaces before inserting links?

Reply:
Yes that is much simpler, thank you!